site/en/blog/chromium-chronicle-28/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "The Chromium Chronicle #28: Getting started with Bling"
+description: >
+  Learn how to get started with Bling, Chrome on iOS.
+layout: 'layouts/blog-post.njk'
+date: 2022-01-25
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/hgu6uTktp2ipmuODZZhP.jpg'
+alt: >
+  Chromium Chronicle image
+tags:
+  - chromium-chronicle
+---
+
+**Episode 28:** by Mark Cogan in Paris, France (January, 2022)<br>
+[Previous episodes](/tags/chromium-chronicle/)
+
+**Bling is Chrome on iOS**. In 2022, Bling will celebrate its 10th anniversary. 
+Chrome is one of the most widely used iOS apps—it's currently the number 2 Utility app in the US App Store. 
+
+Bling provides all of the great Chrome features our users love, including Sync, Translate, Incognito, Password Manager, Autofill, and many more. 
+Bling also integrates with native iOS features such as multi-window and device-wide password autofill. 
+
+However, there's one major thing that makes Bling very different from all other Chrome platform implementations.
+
+Bling doesn't use Blink. 
+
+Because of Apple's App Store rules, Bling has to use the iOS platform API's for fetching and rendering web content. So Bling uses WebKit, Nitro and CFNetwork where other Chrome platforms are using Blink, V8 and `//net`. Due to this, and the limited API available for iOS apps to interact with web page content, many Chrome features that use the `//content` APIs are implemented in Bling using JavaScript injection. 
```

Comment:
```suggestion
Because of Apple's App Store rules, Bling has to use the iOS platform APIs for fetching and rendering web content. So Bling uses WebKit, Nitro, and CFNetwork, while other Chrome platforms use Blink, V8, and `//net`. Due to this, and the limited APIs available for iOS apps to interact with web page content, many Chrome features that use the `//content` APIs. The `//content` APIs are implemented in Bling through JavaScript injection. 
```

Reply:
this has changed the meaning of the final sentence.

Reply:
Yes, this is now incorrect. iOS Chrome doesn't implement the //content APIs at all. Some features on other Chrome platforms use the //content APIs (for example, autofill). iOS Chrome cannot use the //content APIs. So to build these features, iOS Chrome uses JavaScript injection instead.

Also the penultimate sentence of the paragraph is syntactically incorrect after this edit (there's no longer a verb that matches the "Due to this..." at the beginning).